customers/migrations/0004_change_boat_weight_to_int.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.6 on 2020-01-09 13:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("customers", "0003_add_validators_to_boat_dimensions"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="boat",
+            name="weight",
+            field=models.PositiveIntegerField(
```

Comment:
Will this automatically get rid possible decimals i.e. is it safe to do without a data migration or do we even need to care about the data at this point?

Reply:
we do not have any data at this point, so I just skipped data migration.